daemon/loadbalancer.go

```diff
@@ -426,6 +427,40 @@ func (d *Daemon) RevNATDump() ([]loadbalancer.L3n4AddrID, error) {
 	return dump, nil
 }
 
+func openServiceMaps() error {
+	if option.Config.EnableIPv6 {
+		if _, err := lbmap.Service6Map.OpenOrCreate(); err != nil {
+			return err
+		}
+		if _, err := lbmap.RevNat6Map.OpenOrCreate(); err != nil {
+			return err
+		}
+		if _, err := lbmap.RRSeq6Map.OpenOrCreate(); err != nil {
+			return err
+		}
+		if _, err := proxymap.Proxy6Map.OpenOrCreate(); err != nil {
+			return err
+		}
+	}
+
+	if option.Config.EnableIPv4 {
+		if _, err := lbmap.Service4Map.OpenOrCreate(); err != nil {
+			return err
+		}
+		if _, err := lbmap.RevNat4Map.OpenOrCreate(); err != nil {
+			return err
+		}
+		if _, err := lbmap.RRSeq4Map.OpenOrCreate(); err != nil {
+			return err
+		}
+		if _, err := proxymap.Proxy4Map.OpenOrCreate(); err != nil {
```

Comment:
Only slight nit I have here is that proxy maps are unrelated to the loadbalancer code, but I think having this code in a separate function is better than how it is currently in master.